container-storage-setup.sh

```diff
@@ -1237,6 +1255,11 @@ if [ -e ${_STORAGE_IN_FILE} ]; then
   source ${_STORAGE_IN_FILE}
 fi
 
+# Error out if CONFIG_NAME is missing.
```

Comment:
@shishir-a412ed My patch got merged where I moved the call of check_storage_options()  So your check should go in there.